src/ParseObject.js

```diff
@@ -1821,15 +1834,15 @@ class ParseObject {
    *
    * @param {String} name Name of Pin.
    * @param {Array} objects A list of <code>Parse.Object</code>.
+   * @return {Promise} A promise that is fulfilled when the pin completes.
    * @static
    */
-  static async pinAllWithName(name: string, objects: Array<ParseObject>): Promise {
+  static pinAllWithName(name: string, objects: Array<ParseObject>): Promise {
```

Comment:
Array<Parse.Object>, right?

Reply:
Parse.Object doesn't exist only the current class `ParseObject`